includes/Modules/Analytics_4.php

```diff
@@ -331,6 +332,13 @@ public function on_deactivation() {
 	public function get_debug_fields() {
 		$settings = $this->get_settings()->get();
 
+		$adsense_connected = true;
+		$adsense_settings  = ( new AdSense_Settings( $this->options ) )->get();
+
+		if ( empty( $adsense_settings['accountSetupComplete'] ) || empty( $adsense_settings['siteSetupComplete'] ) ) {
+			$adsense_connected = false;
+		}
+
```

Comment:
Let's move this to its own method that returns a boolean, e.g. `is_adsense_connected`. What do you think?